src/gc/env/gcenv.os.h

```diff
@@ -128,6 +128,9 @@ typedef void (*GCThreadFunction)(void* param);
 // Interface that the GC uses to invoke OS specific functionality
 class GCToOSInterface
 {
+protected:
+    static size_t s_pageSize;
```

Comment:
@janvorli I did this to address @jkotas request that GCToOSInterface::GetPageSize() be inlined to address his performance concerns.